mikelpardo/mobile-sales
# Add mobile to cart functionality

**Description**

Get mobile detail from given endpoint

**How to test**

1. Go to: http://localhost:3000/
2. Click in one mobile item
3. Select mobile data options and click "add to cart" button
4. Check in the console the log of the response
